src/main/cc/proxy_client/javac_remote_actions.cc

```diff
@@ -0,0 +1,44 @@
+#include "src/main/cc/proxy_client/javac_remote_actions.h"
+
+#include <algorithm>
+
+#include "absl/strings/str_split.h"
+
+namespace remote_client {
+
+const int kMaxArgsToCheckForJavac = 8;
+
+const char* kJavacWrapperCommand = "javac_wrapper ";
+const char* kJavacCommand = "javac ";
+
+using std::min;
+
+void DetermineJavacActionInputs(int argc, char** argv, set<string>* inputs) {
+  for (int i = 0; i < argc; ++i) {
+    string cur(argv[i]);
+    if (cur[0] == '@') {
```

Comment:
As discussed offline, would be nice to get the rsp file from ninja as a followup. It looks like ninja is aware of it.

Reply:
Yup, I'll followup on this separately.

Reply:
Actually done. Removed the rsp file determination and added it to ninja.